addons/crm/tests/test_crm_lead_lost.py

```diff
@@ -18,29 +19,96 @@ def setUpClass(cls):
 
     @users('user_sales_salesman')
     def test_lead_lost(self):
+        """ Test setting a lead as lost using the wizard. Also check that an
+        'html editor' void content used as feedback is not logged on the lead. """
+        # Initial data
+        self.assertEqual(len(self.lead_1.message_ids), 1, 'Should contain creation message')
+        creation_message = self.lead_1.message_ids[0]
+        self.assertEqual(creation_message.subtype_id, self.env.ref('crm.mt_lead_create'))
+
+        # Update responsible as ACLs is "own only" for user_sales_salesman
         self.lead_1.with_user(self.user_sales_manager).write({
             'user_id': self.user_sales_salesman.id,
             'probability': 32,
         })
+        self.flush_tracking()
```

Comment:
Should this be in the "flush the world" commit?
A bit random that it appears here.

Reply:
Well, yes and no. There are more additions to that test... but ok, added the flush + message number part in the previous commit, and re-updated its message. Hopefully it is clearer now :) .